addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/service/NetworkUtils.java

```diff
@@ -167,4 +167,25 @@ public static boolean nativePing(String hostname, int port, int timeout)
         return exitValue == 0;
     }
 
+    public static boolean arping(String hostname, int timeout, String networkInterface)
+            throws InvalidConfigurationException, IOException, InterruptedException {
+        Process proc;
+        if (SystemUtils.IS_OS_UNIX) {
+            proc = new ProcessBuilder("arping", "-I", networkInterface, "-w", String.valueOf(timeout / 1000), "-c", "1",
```

Comment:
From a quick look I'd suggest to enhance [this function](https://github.com/Pakast/openhab2-addons/blob/a9b2b19726293b70a608cd0802f2f08df1c3bd7f/addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/service/NetworkService.java#L118) with a testing command (--version possibly). Set `this.useArping=true` if arping executable else throw exception and set to false.